app/assets/stylesheets/modules/posts.scss

```diff
@@ -4,6 +4,24 @@
   text-align: left;
   width: 70%;
 
+  /*Basic styles for tables*/
+
+  table {
```

Comment:
Amm, tables are very useful for displaying information; in fact, they're more semantic and representative than Ul with Li's splitted ... or what do you mean?